provider/github/internal/builtin/request.go

```diff
@@ -122,18 +144,17 @@ func (r Request) argsToRequest(terms []*ast.Term) (*retryablehttp.Request, error
 
 	u, err := url.Parse(path)
 	if err != nil {
-		return nil, err
+		return "", err
 	}
-
 	u.RawQuery = qs.Encode()
 
-	return r.Client.NewRequest(method, u.String(), nil)
+	return u.String(), err
```

Comment:
I think it's better to return `nil` here instead of `err`. Telling explicitly there was no error (`err` can have a value that we handled and we'd be returning it anyway).